cmd/clusterctl/client/config/reader_viper.go

```diff
@@ -144,7 +147,12 @@ func downloadFile(url string, filepath string) error {
 		Timeout: 30 * time.Second,
```

Comment:
We could move this timeout to the context now, but reading through recommended practices, I think in this case either way works for our needs.